bedrock/mozorg/templates/mozorg/home/home-new.html

```diff
@@ -31,6 +31,9 @@
   {{ css_bundle('protocol-newsletter') }}
   {{ css_bundle('home-new') }}
 
+  {% if LANG == 'de' and switch('de-ctd-homepage-promo') %}
```

Comment:
Suggestion (non-blocking): I think this can be simplified to:

```suggestion
  {% if switch('de-ctd-homepage-promo', ['de']) %}
```